### DISCUSSION TITLE
مشکل در خواندن قیمت صف عرضه

### DISCUSSION THREAD

**Discussion by rezatal** [Glyphack/pytse-client#255] [General]
سلام ممنون بابت کتابخانه خوب تون
در کد زیر خروجی قیمت صف عرضه (best_supply_price) به اشتباه تعداد عرضه کننده های ردیف اول را نمایش میدهد. (در ساعات باز بودن بازار هم این مشکل وجود داشت)

      symbol = 'اهرم'
      ticker = tse.Ticker(symbol)
      df0 = ticker_real_time_data_to_csv(ticker).T

![image](https://github.com/user-attachments/assets/601bd970-40b9-4bb9-8153-1100cf34c941)
![image](https://github.com/user-attachments/assets/e0a9493a-0948-4005-a451-586f6ece0d0d)

**Comment by Glyphack** [comment]
سلام مرسی بابت گزارش فکر کنم این لیستی که میگرفتیم یه سری عدد پشت سر هم بود. متأسفانه سایت بورس برای من کار نمیکنه نمیتونم درستش رو ببینم ولی اگر وقت داشته باشید این بخش از کد رو نگاه کنید احتمالا یکی از عدد ها اونجا ترتیبش عوض شده اخیرا
https://github.com/Glyphack/pytse-client/blob/master/pytse_client/ticker/api_extractors.py#L26
سوالی بود میتونم راهنمایی کنم.
